packages/jitm/src/JITM.php

```diff
@@ -149,6 +153,116 @@ static function jitm_jetpack_woo_services_activate( $CTA ) {
 		);
 	}
 
+	/**
+	 * This is an entire admin notice dedicated to messaging and handling of the case where a user is trying to delete
+	 * the connection owner.
+	 */
+	function delete_user_update_connection_owner_notice() {
+		global $current_screen;
+
+		if ( 'users' !== $current_screen->base ) {
+			return;
+		}
+
+		if ( 'delete' !== $_REQUEST['action'] ) {
```

Comment:
You should check if `isset( $_REQUEST['action'] )` first to avoid _ Undefined index_ PHP notices.